assets/scss/shortcodes/tabbed-pane.scss

```diff
@@ -1,16 +1,12 @@
-.tab-content {
-  .tab-pane {
-    pre {
-      margin: 0rem 0 0rem 0;
-    }
```

Comment:
@deining - setting `pre` margin to 0 doesn't seem necessary given that margin is set to 0 for `.tab-content .tab-pane .highlight` below.

Reply:
Fine for me.